### ISSUE TITLE
Widget shape error handling

### ISSUE THREAD

**Issue by Skrypt** [OrchardCMS/OrchardCore#8969]
Throw a more explicit exception when a Content Type has been removed.

**Comment by deanmarcussen** [comment]
Duplicate ? https://github.com/OrchardCMS/OrchardCore/issues/1014

**Comment by Skrypt** [comment]
Nah, I don't want to create a solution for the issue but just return a proper error message.
